### ISSUE TITLE
Make possible to use markdown elements when using WYSIWYG mode

### ISSUE THREAD

**Issue by ctwhome** [nhn/tui.editor#2961]
## Version
latest from the demos Page
https://nhn.github.io/tui.editor/latest/tutorial-example09-editor-with-color-syntax-plugin

## Current Behavior
<img width="683" alt="image" src="https://user-images.githubusercontent.com/4195550/221668435-a3fb9a1c-f8bd-432e-83d3-358c2b2342b2.png">

## Expected Behavior
It should apply 'heading 1' when writting # Heading 1... and so on. but in WYSIWYG mode doesn't work.

**Comment by ats1999** [comment]
Many editor support this feature i don’t know why tui editor doesn’t support this.

**Comment by ctwhome** [comment]
It does support in in split mode, which is wonderful, but not in the WYSIWYG which is probably where I would expect it more

**Comment by drl990114** [comment]
Yes, I am also troubled by this problem. I think it is necessary to support markdown syntax in WYSIWYG mode.

**Comment by akralj** [comment]
This would be a great feature to have in wysiwyg mode. Can anybody point me where and how this could be possibly implemented please. 
I mainly use -,1., # and ```.
Thanks a lot for this great editor :)
